ui/lib/core/addon/templates/components/box-radio.hbs

```diff
@@ -1,64 +1,66 @@
 {{#if @disabled}}
-<div class="box-radio-spacing">
   <ToolTip
     @verticalPosition="above"
     @horizontalPosition="center"
     as |T|>
     <T.trigger @tabindex="-1">
-      <label
-        for={{@type}}
-        class="box-radio is-disabled is-marginless"
-        data-test-mount-type-radio
-        data-test-mount-type={{@type}}
+      <div
+        class="box-radio is-disabled"
+        data-test-mount-type={{@key}}
+        tabindex="0"
       >
         <Icon
           @glyph={{@glyph}}
           @size="xl"
           class="has-text-grey-light"
         />
-        {{@displayName}}
-        <RadioButton
-          @value={{@type}}
-          @radioClass="radio"
-          @groupValue={{@groupValue}}
-          @changed={{@onRadioChange}}
-          @name={{@groupName}}
-          @radioId={{@type}}
-          @disabled={{@disabled}}
+        <label for={{@key}}>{{@displayName}}</label>
+        <input
+          type="radio"
+          disabled
+          id={{@key}}
+          name={{@groupName}}
+          value={{@key}}
+          tabindex="-1"
```

Comment:
Are we sure changing the native radio group behavior is what you want to do here? I find the default behavior to be strange (you focus the group and then can use arrow keys to navigate through the options like it's one big control), but I don't know if it's surprising to not use the default.

Reply:
Since the input itself is hidden, the default behavior was obscured and led to a confusing interface where tabbing through it looks like nothing is selected. Open to other ways of tackling this though

Reply:
I had this highlighted in my review before I saw this comment. The tabbing vs arrow key functionality seems a little strange to me. I'd assume a disabled input would be skipped without needing to add `tabindex="-1"` here. It seems surprising, in addition, to use tab instead of arrow keys for moving through available options.

Reply:
Ah yeah - is hiding it key to the fix? Was that causing the lag in updating?

I think previously the styling for focus vs not focused wasn't super differentiated:

Focused:
<img width="131" alt="CleanShot 2021-03-15 at 11 40 30@2x" src="https://user-images.githubusercontent.com/39469/111188875-8300a680-8583-11eb-8a01-45136fdf8529.png">

Not focused:
<img width="134" alt="CleanShot 2021-03-15 at 11 40 38@2x" src="https://user-images.githubusercontent.com/39469/111188915-8dbb3b80-8583-11eb-9bff-7133511d4aac.png">

Reply:
I was trying to highlight the box instead of just the mock-input, but with the most recent changes I'm keeping the default radio input because it allows for the arrow (not tab) moving around. 

Thank you all for the discourse! Let me know if the new approach seems unreasonable for any reason